apps/builddao/widget/components/modals/CreateProject.jsx

```diff
@@ -151,10 +195,25 @@ const Main = styled.div`
       background-image: url("data:image/svg+xml,%3csvg xmlns=%27http://www.w3.org/2000/svg%27 viewBox=%27-4 -4 8 8%27%3e%3ccircle r=%273%27 fill=%27%23fff%27/%3e%3c/svg%3e");
     }
   }
+  .location-team {
+    display: flex;
+    gap: 10px;
+    flex-direction: row;
+    @media only screen and (max-width: 768px) {
+      column-gap: 20px;
+      flex-direction: column;
+    }
+  }
+  .err {
+    color: #ff8888;
+    font-size: 12px;
+    padding: 0;
+    margin: 0;
+  }
 `;
 
 function onCreateProject() {
-  const projectAccountId = "testing122.near";
+  const projectAccountId = accountId || "";
```

Comment:
this should be a field in form, prefill that field using signed in account Id, not hardcoded text

Reply:
![Screen Shot 2024-04-16 at 4 29 39 PM](https://github.com/NEARBuilders/gateway/assets/65053264/40c5ae0a-c1dc-4462-8dc6-fa5e18f3c40f)
accountId is not hardcoded though, just taking the currently signed account from context.
I will add the input and prefill